velox/expression/fuzzer/ExpressionFuzzerTest.cpp

```diff
@@ -200,6 +200,7 @@ int main(int argc, char** argv) {
         "combine_hash_internal",
         "map_keys_by_top_n_values", // requires
                                     // https://github.com/prestodb/presto/pull/24570
+        "array_join", // https://github.com/facebookincubator/velox/issues/12452
```

Comment:
This should be separate PR and not part of this change since it would be good to ensure that your changes are being tested in the fuzzer.